tools/internal_ci/linux/grpc_xds_resource_cleanup.sh

```diff
@@ -15,19 +15,18 @@
 
 set -ex
 
+# consts
+readonly GITHUB_REPOSITORY_NAME="grpc"
+readonly TEST_DRIVER_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/${TEST_DRIVER_REPO_OWNER:-grpc}/grpc/${TEST_DRIVER_BRANCH:-master}/tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh"
+
 cd "$(dirname "$0")/../../.."
 
-pyenv local 3.6.1
-gcloud container clusters get-credentials interop-test-psm-sec-v2-us-central1-a --zone us-central1-a --project grpc-testing
+# Source the test driver from the master branch.
+echo "Sourcing test driver install script from: ${TEST_DRIVER_INSTALL_SCRIPT_URL}"
+source /dev/stdin <<< "$(curl -s "${TEST_DRIVER_INSTALL_SCRIPT_URL}")"
+kokoro_setup_test_driver "${GITHUB_REPOSITORY_NAME}"
```

Comment:
Instead of `gcloud container clusters get-credentials ...`, you need to do `activate_gke_cluster GKE_CLUSTER_PSM_SECURITY`

Reply:
Done